### ISSUE TITLE
Used multi-line imports are wrongly marked as unused

### ISSUE THREAD

**Issue by ShetlandJ** [BerniWittmann/vue-unused-components-checker#27]
This type of import can return a false positive:

```javascript
import MyComponent
    from '@/my-path/MyComponent.vue';
```
